### PR TITLE
Add db columns

### PR DESCRIPTION
**Pull request by wrmacdonald**
Added many db columns to user, group, & item

### CODE REVIEW

File: gift-app/database/models/models.py [hunk 1]
```diff
@@ -11,14 +12,21 @@
...
+                      'items.id', 'items.idea')
 
     id = Column(Integer, primary_key=True)
-    name = Column(String(254))
+    first_name = Column(String(254))
     last_name = Column(String(254))
+    email_address = Column(String(254))
+    hashed_password = Column(String(254))
+    time_created = Column(DateTime(timezone=True), server_default=func.now())
+    is_activated = Column(Boolean)
```
Comment (mgnielsen1) [inline gift-app/database/models/models.py]: Can you make isavtive default to False?

File: gift-app/resources.py [hunk 2]
```diff
@@ -81,21 +81,21 @@ def get():
...
         returns:
         - success: 201 and serialized User
         """
 
         try:
             user_post_args = reqparse.RequestParser()
-            user_post_args.add_argument('name', type=str, help='name of the user is required', required=True)
-            user_post_args.add_argument('last_name', type=str, help='last name of the user', required=True)
+            user_post_args.add_argument('first_name', type=str, help='first name of the user is required', required=True)
+            user_post_args.add_argument('last_name', type=str, help='last name of the user is required', required=True)
```
Comment (mgnielsen1) [inline gift-app/resources.py]: I think that email address should be the only required param, can you add it and then make first_name and last_name optional?

**Review by mgnielsen1 — CHANGES_REQUESTED** [review]
Now that we have a test file, can you ensure that is still passes before merging a PR? The tests may need to be adjusted since the use object has changed. You can run them by running "python3 tests/user_tests.py"

File: gift-app/database/models/models.py (hunk 1 above)
Comment (wrmacdonald) [reply gift-app/database/models/models.py]: I think it's fine to have it default to False, but what are we thinking "is_activated" should be used for? My thought is when someone adds emails to invite people to a group, we'll add those emails into the db with is_activated=false. Then when people sign up we'll check for whether their email already exists in the db, & if it does, change is_activated=true while adding all the other info? But when people just sign up and their email isn't in the db yet, we'll add them with is_activated=true. Does that match your idea?

File: gift-app/resources.py (hunk 2 above)
Comment (wrmacdonald) [reply gift-app/resources.py]: 👍🏻

**Comment by mgnielsen1** [comment]
My thought was that activated is always defaults to false and is only set
to true once the user confirms their email. Just thinking now though that
since there are multiple ways of becoming a user there may be a better way
to do it. Maybe having a user.status_code with a few options or something
similar. Let’s open an issue for this to decide the best way to do it.

**Pull request merged**